Orange/base.py

```diff
@@ -86,7 +86,6 @@ class Learner(ReprableWithPreprocessors):
     #: A sequence of data preprocessors to apply on data prior to
     #: fitting the model
     preprocessors = ()
-    learner_adequacy_err_msg = ''
```

Comment:
You can not just delete such a public attribute. This needs to stay and be deprecated if not used anymore.

Reply:
Ooops. In my first review, I incorrectly assumed that @JakaKokosar added this attribute. I'm sorry, Jaka. :(

I still believe that having such attributes is a bad idea. (I hope it wasn't me who added it.) Please, let us deprecate it. Or make it a property.